crates/astria-sequencer-relayer/justfile

```diff
@@ -1,3 +1,5 @@
+set fallback
```

Comment:
this is for when you are not in the root of the repo?

Reply:
Yeah, specifically it says that  this justfile will fallback to any parent justfiles if commands don't exist.

So you can use the build command while in any folder of the repo.